### ISSUE TITLE
Unable to edit a newly created file after saving when using NVDA

### ISSUE THREAD

**Issue by matheuspereira12** [microsoft/vscode#333424]
Does this issue occur when all extensions are disabled?: No

* VS Code Version: 1.136.0
* OS Version: Windows 11

Steps to Reproduce:

1. Open VS Code with NVDA running.
2. Create a new file.
3. Enter a file name and extension, then save the file.
4. Try to edit the file after saving it.
5. The file does not respond to editing as expected.
6. Closing and reopening VS Code makes the file editable again.

Expected Behavior:

The newly created file should remain editable immediately after saving, without requiring VS Code to be closed and reopened.

Actual Behavior:

After saving a newly created file, I cannot edit it normally. I have to close and reopen VS Code before I can edit the file again.

Additional Information:

I use NVDA as my screen reader. I have not yet tested whether this issue occurs when all extensions are disabled.

**Comment by Shivang9983** [comment]
Hi @maintainer,

I would like to work on this issue! I have reviewed the description and can implement the fix along with necessary tests.

Could you please assign this to me? Thanks!

**Comment by meganrogge** [comment]
I do not have windows to test this and not sure what would cause it... @aiday-mar can you try to reproduce? Do you know of any changes here?

**Comment by aiday-mar** [comment]
Hi @meganrogge I put this on my todo list.
